crates/sui-framework/sources/governance/sui_system.move

```diff
@@ -464,30 +467,30 @@ module sui::sui_system {
     ///    gas coins.
     /// 3. Distribute computation charge to validator stake.
     /// 4. Update all validators.
-    public entry fun advance_epoch(
+    fun advance_epoch(
```

Comment:
One benefit of making this private is that we can now change the signature of this in framework upgrade however we want without breaking compatibility @lxfind